gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/DagActionStoreChangeMonitor.java

```diff
@@ -107,6 +115,37 @@ protected void assignTopicPartitions() {
     return;
   }
 
+  /**
+   * Load all actions from the DagActionStore to process any missed actions during service startup
+   */
+  private void initializeMonitor() {
+    // TODO: exclude the dag actions that have a lease in the arbiter store if this is processing too many events
```

Comment:
Can you digress what you mean by this comment? A bit confused

Reply:
I'll remove this for now unless we see other problems but basically the idea was I can do a join of dag actions with completed leases in the lease arbiter table to exclude actions that have already been completed but it's only an interim soln if this doesn't work and kind of hacky